modules/logging-set-output-rate-limit.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * logging/performance_reliability/logging-flow-control-mechanisms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-set-output-rate-limit_{context}"]
+= Configuring log forwarder output rate limits
+
+You can limit the rate of outbound logs to a specified output by configuring the `ClusterLogForwarder` custom resource (CR).
+
+.Prerequisites
+
+* You have installed the {clo}.
+* You have administrator permissions.
+
+.Procedure
+
+. Add a `maxRecordsPerSecond` limit value to the `ClusterLogForwarder` CR for a specified output.
++
+The following example shows how to configure a per collector output rate limit for a Kafka broker output named `kafka-example`:
++
+.Example `ClusterLogForwarder` CR
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+# ...
+spec:
+# ...
+  outputs:
+    - name: kafka-example <1>
+      type: kafka <2>
+      limit:
+        maxRecordsPerSecond: 10M <3>
```

Comment:
Per https://github.com/openshift/cluster-logging-operator/blob/release-5.8/bundle/manifests/logging.openshift.io_clusterlogforwarders.yaml#L511-L515, `10M` is not a valid value, should be `1000000`.